javascript/app.js

```diff
@@ -166,12 +169,8 @@ FinancialFreedom.controller('TimeToRetirementController', ['$scope', 'Retirement
     
     var retirement_data = RetirementCalculatorService.calculateRetirementInfo();
 
-    $(function () {
-        $('[data-toggle="popover"]').popover();
-    });
-
     $scope.showSteps = false;
-    $scope.penClicked = false;
+    $scope.isCollapsed = true;

```

Comment:
Perhaps call this "editCollapsed" or something, so we know what it's referring to.